docs/tutorials/customSessionKeys/createSession.md

```diff
@@ -0,0 +1,253 @@
+---
+sidebar_label: 'Session Module'
+sidebar_position: 5
+---
+# Session Module
+
+In this guide, we will walk through creating a basic Node.js script using
+TypeScript that allows user to create a session.
+
+:::info This tutorial has a previous other steps in the previous sections:
+[Environment Setup](environmentsetup) and
+[Initializing Account](initializeaccount)
+ :::
+
+This tutorial will be done on the Polygon Mumbai Network. We will be using session Module for this.
+
+We will import sessionKeyManagerModule and DEFAULT_SESSION_KEY_MANAGER_MODULE from Biconomy Modules package.
+
+First of all we will initialise the sessionFileStorage using our custom File storage
+
+```typescript
+const sessionFileStorage: SessionFileStorage = new SessionFileStorage(address)
+```
+
+We store the private key in the file using following method. This saves the signer in the file.
+
+```typescript
+const sessionSigner = ethers.Wallet.createRandom();
+const sessionKeyEOA = await sessionSigner.getAddress();
+await sessionFileStorage.addSigner(sessionSigner)
```

Comment:
did you check if we really need to do this step? because it's done from internal method already

Reply:
we need to this explictly.

Reply:
since yuo already have the signer on client side it should not be required to add and get signer.